app/helpers/shift_commerce/cache_helper.rb

```diff
@@ -6,6 +6,10 @@ def shift_cache object, *args
     end
 
     def menus_cache(reference)
+      # Dont fetch from cache if requested for a preview
+      return yield if params[:preview].present? && params[:preview] === 'true'
```

Comment:
do you need `params[:preview].present?`? thought if it's not there it would return nil so `params[:preview] === 'true'` would fail?